### DISCUSSION TITLE
Arch Network

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#5464] [General]
Hello @0xfen I just got off the phone with Amine and he gave me your GH handle because you're the person working with rust-libp2p at arch.network. I'd like to get to know you better and invite you to attend our next rust-libp2p open maintainers call so we can get to know you better.

Cheers!

**Comment by 0xfen** [comment]
Hi @dhuseby, sure I'd like to catch up with you too and also looking forward to the next rust-libp2p open maintainers call.
